src/components/Header/styles.module.scss

```diff
@@ -0,0 +1,75 @@
+@import 'scss/fonts/_fonts';
+@import 'scss/variables/_colors';
+@import 'scss/variables/_content';
+@import 'scss/margins';
+
+.content-header {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  max-width: $max-content;
+  padding: 15px;
+
+  .logo {
+    color: $greenFern;
+    cursor: pointer;
+    font-family: $Titillium;
+    font-size: 30px;
+    font-weight: bold;
+  }
+
+  .list-items {
+    display: flex;
+    justify-content: flex-end;
+
+    .item {
+      margin: 0 10px;
+
+      .link {
+        align-items: center;
+        display: flex;
+        font-size: 18px;
+        justify-content: flex-start;
+        opacity: 0.5;
+
+        .text {
+          color: $black;
+          font-size: 18px;
+        }
+
+        &:hover {
+          opacity: 0.7;
+        }
+
+        .icon {
+          margin-right: 3px;
+        }
+      }
+
+      .active {
+        color: $greenApple;
+        font-weight: bold;
+        opacity: 1;
+
+        .text {
+          color: $greenApple;
+          font-weight: bold;
+        }
+      }
+    }
+  }
+}
+
+@media (max-width: $breakpoint-tablet) {
+  .content-header {
+    .logo {
+      font-size: 30px;
+    }
+
+    .link {
+      font-size: 30px !important;
```

Comment:
Can't avoid using the important property?
You can add specificity here.

Reply:
Yes XD